admin-dev/themes/new-theme/template/components/layout/quick_access.tpl

```diff
@@ -4,30 +4,22 @@
     <span class="selected-item">{l s='Quick Access'}</span> <i class="material-icons arrow-down">keyboard_arrow_down</i>
   </span>
   <div class="ps-dropdown-menu dropdown-menu" aria-labelledby="quick-access">
-    <ul class="items-list">
-      {foreach $quick_access as $quick}
-      <li><a href="{$baseAdminUrl}{$quick.link|escape:'html':'UTF-8'}" class="dropdown-item" data-item="{$quick.name}">{$quick.name}</a></li>
-      {/foreach}
-    </ul>
+    {foreach $quick_access as $quick}
+      <a href="{$baseAdminUrl}{$quick.link|escape:'html':'UTF-8'}" class="dropdown-item" data-item="{$quick.name}">{$quick.name}</a></li>
+    {/foreach}
     <hr>
-    <ul class="items-list">
-      {if isset($matchQuickLink)}
-        <li>
-          <a class="dropdown-item js-quick-link" data-method="remove" data-quicklink-id="{$matchQuickLink}" data-rand="{1|rand:200}" data-icon="{$quick_access_current_link_icon}" data-url="{$link->getQuickLink($smarty.server['REQUEST_URI'])}" data-post-link="{$link->getAdminLink('AdminQuickAccesses')}" data-prompt-text="{l s='Please name this shortcut:' js=1}" data-link="{$quick_access_current_link_name|truncate:32}">
-            {l s='Remove from QuickAccess'}
-          </a>
-        </li>
-      {/if}
-      <li {if isset($matchQuickLink)}class="hide"{/if}>
-        <a class="dropdown-item js-quick-link" data-rand="{1|rand:200}" data-icon="{$quick_access_current_link_icon}" data-method="add" data-url="{$link->getQuickLink($smarty.server['REQUEST_URI'])}" data-post-link="{$link->getAdminLink('AdminQuickAccesses')}" data-prompt-text="{l s='Please name this shortcut:' js=1}" data-link="{$quick_access_current_link_name|truncate:32}">
-          {l s='Add current page to QuickAccess'}
-        </a>
-      </li>
-      <li>
-        <a class="dropdown-item" href="{$link->getAdminLink("AdminQuickAccesses")|addslashes}">
-          {l s='Manage quick accesses'}
-        </a>
-      </li>
-    </ul>
+    {if isset($matchQuickLink)}
+      <a class="dropdown-item js-quick-link" data-method="remove" data-quicklink-id="{$matchQuickLink}" data-rand="{1|rand:200}" data-icon="{$quick_access_current_link_icon}" data-url="{$link->getQuickLink($smarty.server['REQUEST_URI'])}" data-post-link="{$link->getAdminLink('AdminQuickAccesses')}" data-prompt-text="{l s='Please name this shortcut:' js=1}" data-link="{$quick_access_current_link_name|truncate:32}">
```

Comment:
Can you avoid too long lines (120 chars) by using multiples lines following this: http://codeguide.co/#html-attribute-order ?
(So not just here)

```html
<a
    class=""
    data-method=""
>
```

Thank you!